staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -257,6 +257,9 @@ type Config struct {
 
 	// StorageVersionManager holds the storage versions of the API resources installed by this server.
 	StorageVersionManager storageversion.Manager
+
+	// ShouldDisableCompressionCheck verifies if the request properties qualifies it for disabling compression.
```

Comment:
verifies is a big word. Better: ... returns whether compression should be disabled for a given request.

Reply:
Done